Victoradukwu/More-Recipes
# #154499568 create components for signup and landing pages

# What does this PR do?
- Create the Layout page and routing
- Create the landing page with top recipes by upvotes
- Create application sign up and login functionalities
#### Description of Task to be completed?
- Create the LayoutPage, DashboardPage, SignInPage, Signup, HomePage, 404Page;
- Set up routing on the LayoutPage component
- Execute recipe fetch for to recipes on the landing page
- Implement the functionalities of user signing and signup
#### How should this be manually tested?
- Start the app with the command `npm run start:dev`
- Go localhost:8000 to see the landing page
- Go localhost:8000/signup to register
- Go localhost:8000/signin to log in
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
feature/154499568/create-componets-for-signup-and-landing-pages
#### Screenshots (if appropriate)

#### Questions:
N/A
